### DISCUSSION TITLE
Bluetooth keyboard not working

### DISCUSSION THREAD

**Discussion by bdach** [ppy/osu#25818] [Q&A] [answered]
Forwarding this from a twitch chat report - user saying their bluetooth keyboard isn't working for textboxes.

https://www.youtube.com/watch?v=c4qSqmV2Cgg

Dunno if it will be enough to reproduce. Probably going to need someone who owns iOS devices to look (@peppy? @frenzibyte?)

**Comment by peppy** [comment, marked as answer]
I think it's this one https://github.com/ppy/osu/issues/23290

**Reply by bdach** [reply]
Reporter said it's been going on for "about 8 months", so that tracks.
